docs/_api-mocking/mock_options.md

```diff
@@ -28,23 +28,32 @@ parameters:
       - Headers
     content: |-
       Match only requests that have these headers set
-    examples: 
+    examples:
       - |-
         {"Accepts": "text/html"}
+  - name: body
+    types:
+      - Object
+      - Headers
+    content: |-
+      Match only requests that have this body (requires a `Content-Type` request header to be set to `application/json`)
+    examples:
+      - |-
+        {"JSON": "body"}
```

Comment:
Not sure what properties are better to use as an example, but in my experience using weighted terms like 'JSON' and 'body' leads to confusion i.e. 'So do I need to include a 'JSON' property that wraps my body?'

Reply:
Ah yes, that’s a good point it’s not very clear. Will change